index.md

```diff
@@ -1,213 +1,270 @@
 ---
-title: Welcome to Flutter
-sidebar: home_sidebar
+title: Easy and fast SDK for beautiful apps
 layout: page
+hide_title: true
 ---
 
-{% include note.html content="_Flutter is an early-stage open-source project_. We hope you try Flutter out and send us [feedback](#get-support)." %}
+<div class="homepage">
+
+    <section class="homepage__key_points">
+        <h1 class="homepage__title">
+        Flutter makes it easy
+        and fast to build
+        beautiful mobile apps.
+        </h1>
+
+        <div class="homepage__tagline">
+        Flutter is a new mobile app SDK
+        to help
+        developers and designers build
+        modern mobile apps for iOS and Android.
+        </div>
+
+        <div class="homepage__button_row">
+        <a href="/getting-started/" class="get-started-button">Get Started</a>
+        {% include note.html content="_Flutter is an alpha, open-source project_." %}
+        </div>
+
+        <div class="key-points">
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Deliver features faster</div>
+
+                <p>
+                Refresh times so fast, you can "paint" your app to life
+                on hardware, emulators, and simulators.
+                </p>
+            </div>
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Craft beautiful UIs</div>
+
+                <p>
+                Delight your users and make your brand stand out with
+                rich motion, smooth scrolling, and
+                beautiful customizable components.
+                </p>
+            </div>
+
+            <div class="homepage__key_point">
+                <div class="homepage__key_point_title">Used by Google</div>
+
+                <p>
+                Flutter is used by Google, works with Firebase
```

Comment:
Maybe it's worth saying Flutter is used by Google and others? I'm a little worried that some people wonder if it's used by Google _only_.